src/utils/minio_crud.py

```diff
@@ -0,0 +1,55 @@
+import os
+from minio import Minio
+from minio.error import S3Error
+
+class MinioCRUDClient:
+    def __init__(self, endpoint, access_key, secret_key, secure=True):
+        self.client = Minio(endpoint, access_key=access_key, secret_key=secret_key, secure=secure)
+
+    def create_bucket(self, bucket_name):
+        try:
+            if not self.client.bucket_exists(bucket_name):
+                self.client.make_bucket(bucket_name)
+                
+        except S3Error as exc:
+            return False
+        return True       
```

Comment:
why do we need to return the bool value here? if got an error - let's raise it too

Reply:
updated

Reply:
thanks!